http-ts/src/index.ts

```diff
@@ -107,6 +108,10 @@ export class TypeDBHttpDriver {
         return this.apiPost(`/v1/transactions/${encodeURIComponent(transactionId)}/rollback`, {});
     }
 
+    analyze(transactionId: string, query: string, queryOptions?: QueryOptions): Promise<ApiResponse<AnalyzeResponse>> {
```

Comment:
Shouldn't analyze take a different kind of query options?

Reply:
Good catch.